molten.c

```diff
@@ -113,7 +109,11 @@ static const zend_module_dep molten_deps[] = {
     ZEND_MOD_OPTIONAL("redis")    
     ZEND_MOD_OPTIONAL("mongodb")    
     ZEND_MOD_OPTIONAL("mysqli")    
+#ifdef HAS_PDO
+    ZEND_MOD_REQUIRED("PDO")
```

Comment:
What is the issue ?
It not available at buildtime, it is still optional in #else above